system/doc/top/templates/index.html.src

```diff
@@ -126,62 +125,87 @@ In addition to the documentation here Erlang is described in several recent book
+</p>
 
-<li> You can of course use any editor you like to write Erlang
-programs, but if you use Emacs there exists editing support such as
+<h3>Editors for developing Erlang</h3>
+
+<p>You can of course use any editor you like to write Erlang
+  programs, but it is very helpful to have an editor environment which knows
+  about the language and can provide support for
 indentation, syntax highlighting, electric commands, module name
 verification, comment support including paragraph filling, skeletons,
-tags support and more. See the <a href="#tools#/index.html">
-Tools</a> application for details.  
+  tags support and more.
+  In the Erlang/OTP delivery an Emacs mode for Erlang is included,
+  see the <a href="#tools#/index.html">Tools</a> application for details.
+</p>
+<p>
+  The Language Server Protocol, also known as LSP,
+  is a popular solution which makes it possible to implement the language
+  specifics once and then use it from many different editor frontends which
+  support the
+  Language Server Protocol.
+</p>
+<p>
+  <a href="https://erlang-ls.github.io/">Erlang LS</a> is a language server providing language features for the Erlang programming language. The server works with Emacs, VSCode, Sublime Text 3, Vim, Theia and probably many more text editors and IDEs which adhere to the LSP protocol.
+</p>
 <p>
 There are also Erlang plugins for other code editors
 <a href="http://github.com/vim-erlang">Vim (vim-erlang)</a> ,
 <a href="http://atom.io/packages/language-erlang"> Atom </a> ,
-<a href="http://erlide.org/index.html">Eclipse (ErlIDE)</a> and
 <a href="http://ignatov.github.io/intellij-erlang/">IntelliJ IDEA</a>.
-<li>When developing with Erlang/OTP you usually test your programs
+<a href="http://erlide.org/index.html">Eclipse (ErlIDE)</a>
+</p>
+
+<h3>The interactive shell</h3>
+<p>When developing with Erlang/OTP you usually test your programs
 from the interactive shell (see <a href="getting_started/users_guide.html">
 Getting Started With Erlang</a>) where you can call individual
 functions. There is also a number of tools available, such as the graphical <a
-href="#debugger#/index.html" >Debugger</a> and the <a href="#observer#/index.html" >Observer tool</a> for inspection of system information, ets and mnesia tables  etc.
+									      href="#debugger#/index.html" >Debugger</a> and the <a href="#observer#/index.html" >Observer tool</a> for inspection of system information, ets and mnesia tables  etc.
+  </p>
 <p> Also note that there are some shell features like history list
 (control-p and control-n), in line editing (Emacs key bindings) and
 module and function name completion (tab) if the module is loaded.
-<p>
+</p>
+
+<h3>Documentation</h3>
 
-<li>OpenSource users can ask questions
+<li>On a Unix system you can view the manual pages from the command
+line using
+<pre>
+    % erl -man &lt;module&gt;
+</pre>
+</li>
+<li>You can also get the documentation for functions from inside the interactive shell by using:
```

Comment:
Wouldn't be a good idea to specify that this option is available since OTP 23.0 (Or whenever EEP-0048 was merged)?

Reply:
I will think about it, but note that this page is part of the documentation for a specific version, in this case it will be part of OTP 24.1. So if you read the documentation for OTP 23 or older you will not see this at all.

Reply:
Will use the PR #6745 for all changes to the index.html and will correct the link to the book + will remove some of the content and link to https://www.erlang.org/community instead. adoptingerlang is mentioned there as one of the resources for learning Erlang.